test/unit/MemberRoles/changeAuthorized.js

```diff
@@ -0,0 +1,31 @@
+const { Role } = require('../utils').constants;
+const { expect } = require('chai');
+
+describe('changeAuthorized', function () {
+  it('should change authorized address for the role', async function () {
+    const { memberRoles } = this.contracts;
+    const { defaultSender, governanceContracts } = this.accounts;
+
+    const authorizedAddressBefore = await memberRoles.authorized(Role.AdvisoryBoard);
+    await memberRoles.connect(governanceContracts[0]).changeAuthorized(Role.AdvisoryBoard, defaultSender.address);
```

Comment:
let's use something else other than `defaultSender.address`

When you write tests, look to make the values arbitrary for generality, whatever that thing that you are switching to is. Arbitrary within constraints of course (eg. arbitrary member)